### ISSUE TITLE
ASP.NET MVC 4.5 Razor template compilation error - CS0433

### ISSUE THREAD

**Issue by theplacefordev** [SteveGilham/altcover#45]
I'm trying to collect code coverage using AltCover v4.0.659 of my ASP.NET MVC app targeting .NET v4.5 using "Instrument now, then gather coverage from a running application" scenario. I run AltCover.exe from net45 folder in nupkg using following command arguments: --save --opencover --single + some filters to skip all 3rd party assemblies. Assembly weaving stage was successful. But when I try to run my web app, I see following error during Razor template compilation by ASP.NET:
```
Compilation Error
Description: An error occurred during the compilation of a resource required to service this request. Please review the following specific error details and modify your source code appropriately. 

Compiler Error Message: CS0433: The type 'System.Tuple' exists in both 'c:\Windows\Microsoft.NET\Framework64\v4.0.30319\Temporary ASP.NET Files\root\631ec658\1bdf8089\assembly\dl3\0cf7232c\7de5f968_e993d401\AltCover.Recorder.g.DLL' and 'c:\Windows\Microsoft.NET\Framework64\v4.0.30319\mscorlib.dll'
```
.NET reflector says that AltCover.Recorder.g.DLL really contains type System.Tuple<> (and others from System.* namespace) and not referencing System.Core. I'm not an expert in F#, but I suppose it's because it's targeting .NET v3.5 framework instead of .NET 4+.
Is there any known workaround for this problem?

**Comment by SteveGilham** [comment]
The new release will take a little longer than expected -- I've hit another instance of this class of issue, so will want to analyse and resolve that as part of the fix.

**Comment by SteveGilham** [comment]
Should be fixed in release 4.0.661

**Comment by theplacefordev** [comment]
Tested it with 4.0.661 - Razor template compilation error has gone. Thank you!
